teacher.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'Person'
```

Comment:
- [ ] Please always make sure to double check your app before submission. You are not loading the required file here. It should be `require_relative 'person'` and not `require_relative 'Person'`. Also, correct the same error in the Student class.


![Screenshot from 2022-09-30 11-45-29](https://user-images.githubusercontent.com/63087491/193253824-4c12efc0-445c-45fc-bf4d-9de92ed2628a.png)

Reply:
@FrederickMih!
Thank you for reviewing my code I will change it.